resources/compile.sh

```diff
@@ -133,13 +136,33 @@ function build_release_model {
     local COLOR_FLAG=
   fi
 
-  pushd build
-  mkdir obj
-  # TODO: Consider moving options to tsconfig.json?
-  npx tsc --module commonjs --target es6 --outDir ./obj ../source/model.ts || die "Compile phase 1 failed for $model"
-  node ./obj/model.js $COLOR_FLAG || die "Compile phase 2 failed for $model"
+  local modelInputFilename="$base_model.model.ts"
+  local modelOutputFilename="$base_model.model.js"
+  local modelInputPackageFilename="$base_model.model.kps"
+  local modelOutputPackageFilename="$base_model.model.kmp"
+  local modelInfoFilename="$base_model.model_info"
+
+  pushd source
+
+  # Compile model
+  kmlmc -o "../build/$modelOutputFilename" "./$modelInputFilename" || die "Unable to build .model.js file"
```

Comment:
Would it help the logs to include `$modelInputFilename` if the build fails?
Also for line 151

Reply:
The model name is clearly logged above so it's not an issue :)